akka-http-core/src/test/scala/io/akka/integrationtest/http/HttpModelIntegrationSpec.scala

```diff
@@ -40,7 +40,7 @@ class HttpModelIntegrationSpec extends WordSpec with Matchers with BeforeAndAfte
   implicit val system = ActorSystem(getClass.getSimpleName, testConf)
   import system.dispatcher
 
-  override def afterAll() = system.shutdown()
+  override def afterAll() = system.terminate()
```

Comment:
OH NOES! Now we have a Dr. Who reference in PR comments, but there is not yet one in my book …